### ISSUE TITLE
Voiceover does not announce the link information while navigating the document in Landscape mode in iOS.

### ISSUE THREAD

**Issue by msftedad** [mozilla/pdf.js#19166]
### Attach (recommended) or Link to PDF file

https://mozilla.github.io/pdf.js/web/viewer.html

### Web browser and its version

Safari Version 18.1.1

### Operating system and its version

iPhone 13 Version 18.1.1

### PDF.js version

1.4

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

Yes

### Steps to reproduce the problem

**Pre-Requisites:**

1. Access the Control Center by swiping downward from the upper-right corner of the Home or Lock screen.
2. Turn off Portrait Orientation Lock.
3. Rotate your device to Landscape mode.
4. Open Settings> Accessibility> Voiceover> Turn on Voiceover.

**Repro Steps:**

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Use swipe gestures to move to the 'Tools' option at header of pdf viewer.
3. Activate the 'Open' control using double tap and upload any pdf (Having links in it (e.g.: Responsible Computing User Agreement 5-4-2022 (1) (1).pdf)) from your system.
4. Now you'll have the pdf file, in pdf viewer. change the phone mode from portrait to landscape.
5. Use swipe gestures to move to the links.
6. Verify using swipe gestures whether Voiceover announce the link information while navigating the document in Landscape mode or not.

### What is the expected behavior?

In Landscape mode, voiceover should announce the complete link information such as 'link Securing Information Systems, when user navigates using swipe gestures in iOS.

### What went wrong?

Voiceover does not announce the link information while navigating the document in Landscape mode in iOS.
In landscape mode voice over announces only 'Link' on each link ('Securing Information Systems link, information security awareness training...) present on PDF using swipe gestures.

https://github.com/user-attachments/assets/b8e652d6-bbcf-4108-add4-44d7a5841174

**Comment by Snuffleupagus** [comment]
Unfortunately you've left out various *required* information, and not filled out the issue template correctly, hence we cannot help. Please take more care when reporting issues to avoid wasting our time!

> ### Attach (recommended) or Link to PDF file
> 
> https://mozilla.github.io/pdf.js/web/viewer.html

That's clearly not a *PDF file*, as this heading asks for.

> ### Web browser and its version
> 
> Safari Version 18.1.1

As outlined in https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support, support for Safari has always been *limited* and furthermore bugs only affecting Safari is generally not fixed.

> ### PDF.js version
> 
> 1.4

That version number has never existed, hence this makes no sense.

> ### Steps to reproduce the problem
> 
> [...]
> 3. Activate the 'Open' control using double tap and upload any pdf (Having links in it (e.g.: Responsible Computing User Agreement 5-4-2022 (1) (1).pdf)) from your system.

Given that you didn't attach this document to the issue, despite the issue template explicitly asking for that, this issue is completely non-actionable.
